merlin/models/tf/distributed/embedding.py

```diff
@@ -0,0 +1,202 @@
+from typing import Union
+
+import sparse_operation_kit as sok
```

Comment:
Maybe we should wrap this in a try/catch so we can throw an better error when it's not there